From df632f5a0be37267bfae0a90de876f1e2969faee Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:21 -0400
Subject: [tools] perf: Remove unused test-volatile-register-var.c

Message-id: <1408953210-26343-254-git-send-email-jolsa@redhat.com>
Patchwork-id: 88990
O-Subject: [PATCH RHEL7.1 BZ1133083 253/322] perf tools: Remove unused test-volatile-register-var.c
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 26f7f9877234e6b9ed87eff4ca450631bafe0182
Author: Yann Droneaud <ydroneaud@opteya.com>
Date: Sat Jan 11 18:48:15 2014 +0100

description
===========
Since commit 01287e2cb7ad, test-volatile-register-var.c is no more built
as part of the automatic feature check.

This patch remove the unneeded file.
---
 delete mode 100644 tools/perf/config/feature-checks/test-volatile-register-var.c

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/feature-checks/test-volatile-register-var.c b/tools/perf/config/feature-checks/test-volatile-register-var.c
deleted file mode 100644
index c9f398d..0000000
--- a/tools/perf/config/feature-checks/test-volatile-register-var.c
+++ /dev/null
@@ -1,6 +0,0 @@
-#include <stdio.h>
-
-int main(void)
-{
- return puts("hi");
-}
-- 
1.7.1